sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1107,6 +1115,41 @@ Exceptions:
 
 |===
 
+==== New Event Member Functions
+
+Table {counter: tableNumber}. Additional member functions of the `sycl::event` class.
+[cols="2a,a"]
+|===
+|Member function|Description
+
+|
+[source,c++]
+----
+template <typename Param>
+typename detail::is_event_profiling_info_desc<Param>::return_type
+event::get_profiling_info(ext::oneapi::experimental::node Node) const;
+----
+
+| Queries the proliling information of a SYCL Graph node for the graph
+  execution associated with this SYCL event. If the requested info is
+  not available when this member function is called due to incompletion of
+  command groups associated with the event, then the call to this member
+  function will block until the requested info is available.
+
+Parameters:
+
+* `Node` - handle to the node for which the profiling information is queried.
+
+Exceptions:
+
```

Comment:
Might also need an error if `Node` is not associated to the graph that was executed by `event`.